wixcode-users/wix-users.service.json

```diff
@@ -916,9 +1200,142 @@
+                      " *   },",
+                      " *   \"createdDate\" : 2020-07-20T12:33:09.775Z",
+                      " * }",
+                      " */" ],
+                  "extra":
+                    {  } } ],
+            "extra":
+              {  } },
+        "members":
+          [ { "name": "essential",
+              "type": "boolean",
+              "doc": "Consent for mandatory cookies for Wix websites, such as for security cookies. Wix places these cookies on your device and these cookies do not require user consent. Always `true`." },
+            { "name": "functional",
+              "type": "boolean",
+              "doc": "Consent for cookies placed on the user's device that \"remember\" user settings to improve user experience. For example, an indication that the user dismissed a popup. The default is `true`." },
+            { "name": "analytics",
+              "type": "boolean",
+              "doc": "Consent for cookies used for analytics, such as Wix analytics, Google Analytics, Yandex Metrica and so on. The default is `true`." },
+            { "name": "advertising",
+              "type": "boolean",
+              "doc": "Consent for cookies used for advertising purposes. This includes third-party scripts and pixels that may potentially place advertising cookies on the device (such as Twitter page view and Facebook Pixel). The default is `true`." },
+            { "name": "dataToThirdParty",
+              "type": "boolean",
+              "doc": "Consent for a user's personal data to be transferred to a third party (like Google Analytics, Facebook Pixel, and FullStory). The default is `true`." } ],
+        "extra":
+          {  },
+        "labels": [] },
+      { "name": "PolicyDetails",
+        "locations":
+          [ { "lineno": 648,
+              "filename": "users.js" } ],
+        "docs":
+          { "summary": "The complete details of the current user's consent policy.",
+            "description":
+              [ "An object containing all the details about the consent policy of the user ",
+                " currently viewing the site. This is returned in the Promise of the `setConsentPolicy()` function." ],
+            "links": [],
+            "examples": [],
+            "extra":
+              {  } },
+        "members":
+          [ { "name": "defaultPolicy",
+              "type": "boolean",
+              "doc": "Whether the policy is the default consent policy set by the site owner. If true, either the user has not set a policy or the site owner has reset the policy." },
+            { "name": "policy",
+              "type": "wix-users.Policy",
+              "doc": "The object representing the user's current consent policy." },
+            { "name": "createdDate",
+              "type": "Date",
+              "doc": "If a cookie exists in the browser defining the current consent policy, the day of the year the policy was set. Otherwise, undefined.",
```

Comment:
I didn't understand this. What is the day of the year? From 1 to 365? Also what does that mean that there's a cookie that defines the consent policy. Doesn't the consent policy define which cookies are being used?

Reply:
Oh I see from the example that it's just the regular date. So maybe just write the date the policy was set.